rationale_net/train/train.py

```diff
@@ -214,6 +214,9 @@ def run_epoch(data_loader, train_model, model, gen, optimizer, step, args):
 
         logit, _ = model(x_indx, mask=mask)
 
+        if args.use_as_tagger == True:
```

Comment:
you don't need to say `== True`, 
just say
``` if args.use_as_tagger```

Reply:
resolved.